.github/workflows/check.yml

```
@@ -0,0 +1,90 @@
+name: Check # 빼먹을 경우 그냥 이 파일 경로가 출력됨. 없앨 수 없나..
```

please remove comment
```suggestion
name: Check
```